frontends/pytorch/python/torch_mlir_utils/codegen/torch_ods_gen.py

```diff
@@ -436,6 +436,7 @@ def emit_with_mutating_variants(key, **kwargs):
         for key in [
                 "aten::tanh : (Tensor) -> (Tensor)",
                 "aten::relu : (Tensor) -> (Tensor)",
+                "aten::sigmoid : (Tensor) -> (Tensor)",
```

Comment:
After making this change you need to run ./build_tools/update_torch_ods.sh to update the `Dialect/Torch/IR/GeneratedAtenOps.td` and check in changes in that file as well.